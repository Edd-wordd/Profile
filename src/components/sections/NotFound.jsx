import React from 'react'
import { useStyles } from '../styles/sections/NotFound.styles'

function NotFound() {
  const classes = useStyles()
  return (
    <div className={classes.errorWrapper}>
      <h1>404</h1>
      <p>Page not found</p>
    </div>
  )
}

export default NotFound
