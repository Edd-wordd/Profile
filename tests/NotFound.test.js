import React from 'react'
import testComponent from './testComponent'
import NotFound from '../src/components/sections/NotFound'

testComponent(<NotFound />, 'NotFound')
