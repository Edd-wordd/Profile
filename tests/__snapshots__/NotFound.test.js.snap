// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render NotFound 1`] = `"Component"`;
